examples/kubernetes/configmap-sidecar-test.yaml

```diff
@@ -10,12 +10,34 @@ metadata:
 data:
   test1: |
     name: test1
-    environment:
```

Comment:
oops, thanks for catching that! I missed that in the refactor before opensourcing